src/index.js

```diff
@@ -24,6 +24,7 @@ const { flyvningstyper } = require('./flyvningstyper');
 const { medlemskapstyper } = require('./medlemskapstyper');
 const { mottaksretning } = require('./mottaksretning');
 const { oppgavetyper } = require('./oppgavetyper');
+const { personstatuser } = require('./personstatuser');
```

Comment:
`personstatuser` må legges til objektet `KodeTermObjects` for å eksporteres for frontend. Se linje 45-82.